src/lager_graylog_gelf_formatter_behaviour.erl

```diff
@@ -0,0 +1,9 @@
+-module(lager_graylog_gelf_formatter_behaviour).
```

Comment:
```suggestion
-module(lager_graylog_formatter).
```

I'd rename the behaviour since not all messages sent to Graylog need to be formatted as GELF.

Reply:
Good point, thanks. Fixed. Should I squash commits?

Reply:
@IgorKarymov I'll squash when merging :)